tests/test_preprocessor.py

```diff
@@ -644,8 +644,6 @@ def test_filter_on_flag_column_nothing_left(self):
         flag_dict = {'flag1': [1, 0, 14], 'flag2': ['TD']}
 
         df_new, df_excluded = self.fpp.filter_on_flag_columns(bad_df, flag_dict)
```

Comment:
We could probably remove the assignment here, too, since `filter_on_flag_columns()` should raise a `ValueError`.

Reply:
Which assignment?

Reply:
`df_new` and `df_excluded` are assigned but never used.

Reply:
Ah, I see. Pretty sure `flake8` will catch that so let's take care of that when we are doing the code style review.